api/provider/aws/test_aws/environment_create_test.go

```diff
@@ -257,6 +257,50 @@ func TestEnvironmentCreateDefaults(t *testing.T) {
 	createSGHelper(t, mockAWS, "l0-test-env_id-env", "vpc_id")
 	readSGHelper(mockAWS, "l0-test-env_id-env", "sg_id")
 
+	mockAWS.EC2.EXPECT().
+		AuthorizeSecurityGroupIngress(gomock.Any()).
+		Return(&ec2.AuthorizeSecurityGroupIngressOutput{}, nil)
+
+	mockAWS.ECS.EXPECT().
+		CreateCluster(gomock.Any()).
+		Return(&ecs.CreateClusterOutput{}, nil)
+
+	target := provider.NewEnvironmentProvider(mockAWS.Client(), tagStore, mockConfig)
+	if _, err := target.Create(req); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestEnvironmentCreateDefaults_static(t *testing.T) {
```

Comment:
Is there a dynamic/stateless test?

Reply:
Yeah - the default behavior of `environment create` in this new release will be to create dynamic/stateless/fargate environments, so `TestEnvironmentCreateDefaults()` is the dynamic counterpart to this `TestEnvironmentCreateDefaults_static()` func. Naming things is hard, though - which is better?
- `TestEnvironmentCreateDefaults()`, which indicates calling `environment create` with the minimum required arguments, and which only _implies_ that it creates defaults for dynamic/stateless environments by virtue of dynamic/stateless environments being the default for `environment create`?
- `TestEnvironmentCreateDefaults_dynamic()`, which is explicit about creating the defaults for dynamic/stateless environments, but which doesn't communicate that it's also the generalized default for `environment create`?